CRM/Contact/Page/DedupeFind.php

```diff
@@ -36,6 +36,23 @@ class CRM_Contact_Page_DedupeFind extends CRM_Core_Page_Basic {
   protected $_mainContacts;
   protected $_gid;
   protected $action;
+  /**
+   * Only display selected.
+   *
+   * @var bool
+   */
+  protected $selected;
+
+  /**
+   * Get isSelected value.
+   *
+   * This needs to be an integer of 0 or 1 or NULL for no filter.
```

Comment:
One of the functions this can hit specifically checks for === 1 or === 0 because traditionally '2' has also been used to indicate ... something else. Make sure we are typing correctly

Reply:
Is there still code that needs this variable to be '2' sometimes? Because it looks like the change you made in `CRM_Contact_Page_AJAX::getDedupes()` will prevent it from being anything but 1 or 0.

Reply:
Actually CRM_Contact_Page_AJAX::getDedupes() only uses $selected in one place

```
    if ($selected) {
      $whereClause .= ' AND pn.is_selected = 1';
    }
```